substrate/client/network/src/peer_store.rs

```diff
@@ -39,15 +39,22 @@ use crate::protocol_controller::ProtocolHandle;
 pub const LOG_TARGET: &str = "peerset";
 
 /// We don't accept nodes whose reputation is under this value.
-pub const BANNED_THRESHOLD: i32 = 82 * (i32::MIN / 100);
+pub const BANNED_THRESHOLD: i32 = 71 * (i32::MIN / 100);
```

Comment:
Worth double-checking we don't have this constant hardcoded anywhere in reputation costs.